frontend/src/pages/ActivityReport/Pages/Review/Approver/index.js

```diff
@@ -17,6 +19,17 @@ const Approver = ({
   const { managerNotes, additionalNotes, status } = formData;
   const review = status === REPORT_STATUSES.SUBMITTED || status === REPORT_STATUSES.NEEDS_ACTION;
   const approved = status === REPORT_STATUSES.APPROVED;
+
+  // NOTE: This is only an estimate of which timezone the user is in.
+  // Not guaranteed to be 100% correct but is "good enough"
+  // https://momentjs.com/timezone/docs/#/using-timezones/guessing-user-timezone/
+  const timezone = moment.tz.guess();
+  const time = moment().tz(timezone).format('MM/DD/YYYY [at] h:mm a z');
+  const message = {
```

Comment:
Have to pass this metadata as an object instead of a string like how I was previously doing it since react router doesn't allow passing in anything else besides primitive types?